unity3d/Assets/Swrve/SwrveSDK/SwrveSDKImp.cs

```diff
@@ -1289,6 +1297,9 @@ protected virtual void ProcessCampaigns (Dictionary<string, object> root)
 
     private void LoadResourcesAndCampaigns ()
     {
+        if (!IsAlive ()) {
```

Comment:
This isn't really related to MessageCenter, but this was filling up the console with error logs of calls happening after we've shut down the server, during Unit Tests.

I thought that because it's already in the check function,
https://github.com/Swrve/swrve-unity-sdk/blob/release-3_7/unity3d/Assets/Swrve/SwrveSDK/SwrveSDKImp.cs#L711

That it should be in the actual loading area too.

I can split this into it's own PR if you'd prefer, though